DEVELOPMENT.md

```diff
@@ -163,7 +163,7 @@ To deploy to CloudFoundry submit the following:
 `cf push federalistapp --strategy rolling --vars-file "./.cloudgov/vars/${CF_SPACE}.yml" -f ./cloudgov/manifest.yml`
 
 ### Continuous Integration
-We are in the process of migrating from CircleCI to an internal instance of Concourse CI, starting with our staging environment. To use Concourse, one must have appropriate permissions in UAA as administered by the cloud.gov operators. Access to Concourse also requires using the GSA VPN.
+Our continuous integration pipeline is run on Concourse CI. To use Concourse, one must have appropriate permissions in UAA as administered by the cloud.gov operators. Access to Concourse also requires using the GSA VPN.
```

Comment:
> Access to Concourse also requires using the GSA VPN.

Maybe it's beyond the scope of this documentation, but Zscaler changes this, or is at least in the process of doing so.